### PR TITLE
Turn 'Open Source' footer message into a link.

### PR DESCRIPTION
**Pull request by lmatiolis**
The idea is just to make the catarse code more accessible, which could end up bringing more contributors.

Ps: Also added a mixin to share style between footer links.

### CODE REVIEW

File: app/views/catarse_bootstrap/shared/_footer_big.html.slim [hunk 1]
```diff
@@ -21,6 +21,7 @@ footer.main-footer.main-footer-neg
...
     .footer-full-copyleft 
       = image_tag "logo-footer.png", class: "u-marginbottom-20"
-      .lineheight-loose Feito com amor | 2015 | Open source
-      
-      
\ No newline at end of file
+      .lineheight-loose
+        |  Feito com amor |
+        |  2015 |
+        a.link-footer-inline< href="#{CatarseSettings[:github_url]}" Open source
```
Comment (lmatiolis) [inline app/views/catarse_bootstrap/shared/_footer_big.html.slim]: I'm not used to Slim, so if there's a better to write this inline link tag with the text before it, please feel free to suggest the change and i'll do it! ;)

File: app/assets/stylesheets/catarse_bootstrap/_main.scss [hunk 2]
```diff
@@ -634,15 +634,14 @@ p {
   background-color: #3f4752;
 }
 .link-footer {
+  @include link-footer-base;
```
Comment (diogob) [inline app/assets/stylesheets/catarse_bootstrap/_main.scss]: This file is generated automatically by http://catarse.webflow.io/
So we'd rather have this PR without changes to the CSS

File: app/assets/stylesheets/catarse_bootstrap/catarse.css.sass [hunk 3]
```diff
@@ -3,6 +3,8 @@
 @import catarse_bootstrap/normalize
 
 // The order below DOES matter. Extra should have priority.
+@import catarse_bootstrap/custom_mixins
```
Comment (diogob) [inline app/assets/stylesheets/catarse_bootstrap/catarse.css.sass]: Anything not in the main should go to the extra file, so we can better isolate the CSS generated by hand (not from webflow)

File: app/views/catarse_bootstrap/shared/_footer_big.html.slim (hunk 1 above)
Comment (diogob) [inline app/views/catarse_bootstrap/shared/_footer_big.html.slim]: I would rather use the entire sentence as a link. So we wouldn't need to worry with the link being inline.
```slim
a.link-footer[href="#{CatarseSettings[:github_url]}"] Feito com amor | #{Date.today.year} | Open source
```

**Comment by lmatiolis** [comment]
@diogob I didn't notice the webflow, sorry. I think i prefer the link only on the "Open Source" text, but i changed it as you suggested.

**Comment by diogob** [comment]
Thanks for the PR :+1:

**Pull request merged**